src/components/PlayerView/events.ts

```diff
@@ -67,65 +67,281 @@ import {
+  /**
+   * Event emitted when the player has been muted.
+   */
   onMuted: MutedEvent;
+  /**
+   * Event emitted when the player has been paused.
+   */
   onPaused: PausedEvent;
+  /**
+   * Event mitted when the availability of the Picture in Picture mode changed.
+   * @platform Android
+   */
   onPictureInPictureAvailabilityChanged: PictureInPictureAvailabilityChangedEvent;
+  /**
+   * Event emitted when the player enters Picture in Picture mode.
+   */
   onPictureInPictureEnter: PictureInPictureEnterEvent;
+  /**
+   * Event emitted when the player entered Picture in Picture mode.
+   */
   onPictureInPictureEntered: PictureInPictureEnteredEvent;
+  /**
+   * Event emitted when the player exits Picture in Picture mode.
+   */
   onPictureInPictureExit: PictureInPictureExitEvent;
+  /**
+   * Event emitted when the player exited Picture in Picture mode.
+   */
   onPictureInPictureExited: PictureInPictureExitedEvent;
+  /**
+   * Event emitted when the player received an intention to start/resume playback.
+   */
   onPlay: PlayEvent;
+  /**
+   * Event emitted when the playback of the current media has finished.
+   */
   onPlaybackFinished: PlaybackFinishedEvent;
+  /**
+   * Event emitted when a source is loaded into the player.
+   * Seeking and time shifting are allowed as soon as this event is seen.
+   */
   onPlayerActive: PlayerActiveEvent;
+  /**
+   * Event Emitted when a player error occurred.
+   */
   onPlayerError: PlayerErrorEvent;
+  /**
+   * Event emitted when a player warning occurred.
+   */
   onPlayerWarning: PlayerWarningEvent;
+  /**
+   * Emitted when playback has started.
+   */
   onPlaying: PlayingEvent;
+  /**
+   * Emitted when the player is ready for immediate playback, because initial audio/video
+   * has been downloaded.
+   */
   onReady: ReadyEvent;
+  /**
+   * Event emitted when the player is about to seek to a new position.
+   * Only applies to VoD streams.
+   */
   onSeek: SeekEvent;
+  /**
+   * Event emitted when seeking has finished and data to continue playback is available.
+   * Only applies to VoD streams.
+   */
   onSeeked: SeekedEvent;
+  /**
+   * Event mitted when the player starts time shifting.
+   * Only applies to live streams.
+   */
   onTimeShift: TimeShiftEvent;
+  /**
+   * Event emitted when time shifting has finished and data is available to continue playback.
+   * Only applies to live streams.
+   */
   onTimeShifted: TimeShiftedEvent;
+  /**
+   * Event emitted when the player begins to stall and to buffer due to an empty buffer.
+   */
   onStallStarted: StallStartedEvent;
+  /**
+   * Event emitted when the player ends stalling, due to enough data in the buffer.
+   */
   onStallEnded: StallEndedEvent;
+  /**
+   * Event emitted when a source error occurred.
+   */
   onSourceError: SourceErrorEvent;
+  /**
+   * Event emitted when a new source loading has started.
+   * It doesn't mean that the loading of the new manifest has finished.
```

Comment:
What's the reason for this additional comment? That's not something I would assume when looking at the event.